graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/fetchers/UpdateMutationFetcher.java

```diff
@@ -1,5 +1,7 @@
 package io.stargate.graphql.schema.cqlfirst.dml.fetchers;
 
+import static io.stargate.graphql.schema.cqlfirst.dml.fetchers.TtlFromOptionsExtractor.getTTL;
```

Comment:
Is this needed?

Reply:
same as https://github.com/stargate/stargate/pull/874#discussion_r620786176